kong/plugins/session/migrations/000_base_session.lua

```diff
@@ -0,0 +1,37 @@
+return {
+  postgres = {
+    up = [[
+      CREATE TABLE IF NOT EXISTS sessions(
+        id            uuid,
+        session_id    text UNIQUE,
+        expires       int,
+        data          text,
+        created_at    timestamp without time zone default (CURRENT_TIMESTAMP(0) at time zone 'utc'),
+        ttl           timestamp with time zone,
```

Comment:
It is counter-intuitive to me that created_at and ttl aren't the type of timestamp. I assume we have this pattern elsewhere as well?

Reply:
yea good catch looks like we switched all created_at to `with time zone` so just made them the same.